esp/esdllib/esdl_transformer2.cpp

```diff
@@ -1196,8 +1197,7 @@ void Esdl2Struct::addChildren(Esdl2Transformer *xformer, IEsdlDefObjectIterator
         {
             m_children.append(*obj);
             m_child_map.setValue(obj->queryName(), obj);
-            if (obj->queryEclName())
```

Comment:
Yes it's intentional. I checked with Tony that EclName is not supposed to be used this way. We could make it a separate Jira (fix it before applying the optimization).